File: Unidad 2/Meta 2.2/Python/Biosenosis.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import Ecosistema
from typing import List

class Biosenosis(object):
	def __init__(self, nombre: str):
		self.nombre = nombre
	
	def setNombre(self, nombre: str):
		self.nombre = nombre

	def getNombre(self) -> str:
		return self.nombre

	def __str__(self):
		return "Biosenosis{" + "nombre=" + self.nombre + '}'

File: Unidad 2/Meta 2.2/Java/Biosenosis.java
public class Biosenosis {

    private String nombre;
    private Ecosistema ecosistema;

    public Biosenosis(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }  

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public Ecosistema getEcosistema() {
        return ecosistema;
    }

    public void setEcosistema(Ecosistema ecosistema) {
        this.ecosistema = ecosistema;
    }

    @Override
    public String toString() {
        return "Biosenosis{" + "nombre=" + nombre + '}';
    }

}